lib/ansible/executor/process/worker.py

```diff
@@ -77,7 +79,10 @@ def __init__(self, rslt_q, task_vars, host, task, play_context, loader, variable
                 fileno = sys.stdin.fileno()
                 if fileno is not None:
                     try:
-                        self._new_stdin = os.fdopen(os.dup(fileno))
+                        if PY3: # needed for recognizing Enter in 'pause' module
+                            self._new_stdin = os.fdopen(os.dup(fileno), newline='\r\n')
```

Comment:
@abadger I followed the original ticket and it seems to work as per tests, not sure of what other side effects we should look for